tests/integration/profiling/rule_based_profiler/test_run_profiler_notebook.py

```diff
@@ -155,3 +155,93 @@ def test_run_self_initializing_expectations_notebook(tmp_path):
             "great_expectations/expectations/new_expectation_suite.json",
         ]
         clean_up_test_files(base_dir=base_dir, paths=paths_to_clean_up)
+
+
+def test_run_multibatch_inferred_asset_example(tmp_path):
+    """
+    What does this test and why?
+    One of the resources we have for multi-batch Expectations is a Jupyter notebook that explains/shows the components
+    in code. This test ensures the codepaths and examples described in the Notebook actually run and pass, nbconvert's
+    `preprocess` function.
+    """
+    base_dir: str = file_relative_path(
+        __file__, "../../../test_fixtures/rule_based_profiler/example_notebooks"
+    )
+    notebook_path: str = os.path.join(
+        base_dir, "MultiBatchExample_InferredAssetFileSystemExample.ipynb"
+    )
+    # temporary output notebook for traceback and debugging
+    output_notebook_path: str = os.path.join(
+        tmp_path, "MultiBatchExample_InferredAssetFileSystemExample_executed.ipynb"
+    )
+
+    with open(notebook_path) as f:
+        nb = nbformat.read(f, as_version=4)
+
+    ep = ExecutePreprocessor(timeout=60, kernel_name="python3")
```

Comment:
@Shinnnyshinshin At some point, type hint, please (does not have to be in this PR as long as we do not forget).  Thanks!

Reply:
Done :) Thanks for the catch